assets/app/features/upgrade/controllers/upgrade-nodes.controller.js

```diff
@@ -81,6 +122,21 @@
             if (response.data.errors) {
                 return;
             }
+
+            // init upgrade compute checkbox
+            if (vm.nodesUpgrade.upgradeComputeCheckboxSelected === null) {
+                if (angular.isDefined(response.data.nodes_selected_for_upgrade)) {
+                    vm.nodesUpgrade.upgradeComputeCheckboxSelected = response.data.nodes_selected_for_upgrade === 'all';
+                } else {
+                    vm.nodesUpgrade.upgradeComputeCheckboxSelected = true;
+                }
+            }
+
+            // init compute node postponed flag
+            if (vm.nodesUpgrade.computeNodesPostponedFlag === null) {
```

Comment:
Again, this should simply read the value from `response` and default to `false`.

Reply:
will do